### PR TITLE
changed test to allow for existing consortia

### PR DESCRIPTION
**Pull request by tkah**
* **Problem:**
   1. Tests fail when testing against system with already existing consortia.
* **Solution:**
    1. Specify which consortium is to be deleted/created.
* **Tests:**
   1. Create separate consortium, project
   1. `npm run test:e2e` in coinstac-ui package directory.

### CODE REVIEW

File: packages/coinstac-ui/app/render/components/consortium-card.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default function ConsortiumCard(props) {
 
   if (isOwner) {
     deleteButton = (
-      <Button bsSize="small" bsStyle="danger" onClick={deleteConsortium}>
+      <Button bsSize="small" bsStyle="danger" onClick={deleteConsortium} id={`delete-${label}`}>
```
Comment (swashcap) [inline packages/coinstac-ui/app/render/components/consortium-card.js]: A consortium’s `label` may have whitespace, which means selection by ID could break. Perhaps change to the consortium’s `id`?

**Review by swashcap — CHANGES_REQUESTED** [review]

File: packages/coinstac-ui/app/render/components/consortium-card.js (hunk 1 above)
Comment (tkah) [reply packages/coinstac-ui/app/render/components/consortium-card.js]: The reason I've added these `id` tags is to be able to grab them from WebdriverIO. In the e2e test, I don't have access to the `id` of a given consortium. I have the name I assign to it, though. Would you be alright with using a modified `label` with spaces replaced by dashes?

Comment (swashcap) [reply packages/coinstac-ui/app/render/components/consortium-card.js]: Ah, gotcha. That sounds like a good approach.

**Review by swashcap — APPROVED** [review]

**Comment by swashcap** [comment]
@tkah This looks good. Could you fix the merge conflicts?

**Comment by swashcap** [comment]
🎉 Works great!

**Pull request merged**